recipe/meta.yaml

```diff
@@ -1,18 +1,15 @@
-{% set version = "1.16.2" %}
+{% set version = "1.16.5" %}
 
 package:
   name: automake
   version: {{ version }}
 
 source:
   url: http://ftp.gnu.org/gnu/automake/automake-{{ version }}.tar.gz
-  sha256: b2f361094b410b4acbf4efba7337bdb786335ca09eb2518635a09fb7319ca5c1
-  patches:
-    - 0001-apple-silicon.patch
-    - 0002-apple-silicon.patch
```

Comment:
Dropped per this comment ( https://github.com/conda-forge/automake-feedstock/pull/30#discussion_r677018120 )